tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -258,6 +258,12 @@ static cl::opt<SPIRV::BuiltinFormat> SPIRVBuiltinFormat(
         clEnumValN(SPIRV::BuiltinFormat::Global, "global",
                    "Use globals to represent SPIR-V builtin variables")));
 
+static cl::opt<bool> SPIRVUseOpenCLExtInstructionsForLLVMIntrinsic(
```

Comment:
I feel that this option sounds very generic. It sounds as if every llvm intrinsic is covered under this option. Can you please elaborate?

Thanks

Reply:
Renamed to SPIRVUseOpenCLExtInstructionsForLLVMMargIntrinsic